package.json

```diff
@@ -53,66 +73,8 @@
     "vinyl": "^2.2.1",
     "yargs": "17.7.0"
   },
-  "scripts": {
-    "build": "gulp build",
-    "clean": "gulp clean",
-    "deploy": "gulp deploy",
-    "promote": "gulp promote",
-    "burgr": "gulp burgr:sonarcloud && gulp burgr:sonarqube",
-    "sonarqube": "gulp sonarqube-analysis:sonarqube && gulp sonarqube-analysis:sonarcloud",
-    "format": "prettier --write --list-different \"{commonv5,common,extensions}/**/*.{js,ts}\"",
-    "format-check": "prettier --list-different \"{commonv5,common,extensions}/**/*.{js,ts}\"",
-    "lint": "eslint --ext js,ts --quiet \"{commonv5,common,extensions}/**/*.{js,ts}\"",
-    "lint-report": "eslint --ext js,ts -f json -o eslint-report.json \"{commonv5,common,extensions}/**/*.{js,ts}\"",
-    "test-build": "gulp build:test",
-    "test": "jest",
-    "test-ci": "jest --coverage",
-    "ts-check": "tsc --noEmit",
-    "validate": "npm run lint && npm run ts-check && npm run format-check && npm run test",
-    "validate-ci": "npm run lint-report && npm run ts-check && npm run format-check && npm run test-ci",
-    "cyclonedx-run": "cyclonedx-node",
-    "cyclonedx": "gulp cycloneDx"
-  },
   "engines": {
     "node": ">=6",
     "npm": ">=8"
-  },
-  "jest": {
-    "clearMocks": true,
-    "collectCoverageFrom": [
-      "common/**/*.ts",
-      "commonv5/**/*.ts"
-    ],
-    "coverageDirectory": "coverage",
-    "coverageReporters": [
-      "lcovonly",
-      "text"
-    ],
-    "moduleFileExtensions": [
-      "ts",
-      "js"
-    ],
-    "testPathIgnorePatterns": [
-      "<rootDir>/node_modules",
-      "<rootDir>/config",
-      "<rootDir>/build"
-    ],
-    "testRegex": "(/__tests__/.*|\\-test)\\.ts$",
-    "testEnvironmentOptions": {
-      "url": "http://localhost/"
-    },
-    "transform": {
-      ".(ts)$": "ts-jest"
-    }
-  },
```

Comment:
for my own reference, this jest configuration is now split in each common directory.

One thing to consider is now that the tests run in two directories, that we compute the coverage correctly.